tools/ARIAtools/extractProduct.py

```diff
@@ -570,21 +571,31 @@ def merged_productbbox(metadata_dict, product_dict, workdir='./',
         bbox_file=prods_TOTbbox
 
     # Warp the first scene with the output-bounds defined above
-    # ensure output-bounds are an integer multiple of interferometric grid and adjust if necessary
+    # ensure output-bounds are an integer multiple of interferometric grid
+    # and adjust if necessary
     OG_bounds = list(open_shapefile(bbox_file, 0, 0).bounds)
     arrres = gdal.Open(product_dict[0]['unwrappedPhase'][0])
-    arrres = [abs(arrres.GetGeoTransform()[1]), abs(arrres.GetGeoTransform()[-1])]
-    ds = gdal.Warp('', gdal.BuildVRT('', product_dict[0]['unwrappedPhase'][0]), options=gdal.WarpOptions(format="MEM", \
-        outputBounds = OG_bounds, xRes = arrres[0], yRes = arrres[1], targetAlignedPixels = True, \
-        options = ['NUM_THREADS = %s'%(num_threads)]))
+    arrres = [abs(arrres.GetGeoTransform()[1]),
+              abs(arrres.GetGeoTransform()[-1])]
+    ds = gdal.Warp('',
+                   gdal.BuildVRT('', product_dict[0]['unwrappedPhase'][0]),
```

Comment:
would be good to define the common parameters of gdal.Warp and gdal.BuildVrt beforehand as submit it as kwargs. This could lead in little code clean up.

gdal_warp_kwargs = dict(format='MEM', targetAlignedPixels=True, multithread=True, options=['NUM_THREADS=%s'%(num_threads)]) 

gdal.Warp(**gdal_warp_kwargs)

Reply:
@mgovorcin ok I adjusted all of the gdalwarp option calls in `extractProduct.py`.

There are other scripts/functions I will extend this to, but I think it's more appropriate for a follow-up PR since this one is getting a bit large! Let me know what you think